docs/loading_group_transaction.rst

```diff
@@ -0,0 +1,51 @@
+.. _loading_group_transaction:
+
+Loading Values from Group Transactions
+======================================
+
+Since TEAL version 4 and above, programs can load values from transactions within an atomic 
+group transaction. For instance, you can access the ID or scratch slot of another contract 
+that is being created within an atomic group of transactions. These operations are only 
+permitted in application mode.
+
+Accessing IDs
+~~~~~~~~~~~~~
+
+The ID of an asset or application from another transaction in the current atomic group
```

Comment:
I think a better title for this section would be "Accessing IDs of New Apps and Assets"

Also, could you reword the first sentence to be "The generated ID of an asset or application from a creation transaction in the current atomic group can be accessed..."?